packages/embark-test-runner/package.json

```diff
@@ -52,6 +52,7 @@
     "embark-i18n": "^4.1.1",
     "embark-utils": "^4.1.1",
     "fs-extra": "7.0.1",
+    "istanbul": "0.4.5",
```

Comment:
I have no desire to make more work here, but I'm wondering if the v5 transition is to the time to use more recent versions of the istanbul tech: https://github.com/istanbuljs/nyc/blob/v14.1.1/package.json#L79-L84

Reply:
As I've previously mentioned, the format that these libraries use is now different, so it's a non insignificant amount of work to get this done.